frontend/model/contracts/shared/constants.js

```diff
@@ -32,6 +32,7 @@ export const PAYMENTS_ARCHIVED = 'payments-archived'
 export const MAX_ARCHIVED_PERIODS = 100
 export const MAX_SAVED_PERIODS = 2
 export const MAX_HISTORY_PERIODS = 6
+export const MAX_GROUP_MEMBER_COUNT = 150 // Dunbar's number (https://en.wikipedia.org/wiki/Dunbar's_number)
```

Comment:
we can test this PR by temporarily reducing this constant to a small number like 5 while running this feature branch locally.